tests/Unit/UtilizationMetricsApisTest.php

```diff
@@ -0,0 +1,436 @@
+
+    $licenseSettings->data->limits->conversationalAiSeats = $randomRecords + 1;
+    $licenseSettings->save();
+
+    User::factory()->count($randomRecords)->licensed([LicenseType::ConversationalAi])->create();
+
+    $softDeleteAiUser = User::factory()->licensed([LicenseType::ConversationalAi])->create();
+    $softDeleteAiUser->delete();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['ai_users'])->toBe($randomRecords);
+});
+
+it('checks the API returns AI Exchanges', function () {
+    TrackedEventCount::factory()->create([
+        'type' => TrackedEventType::AiExchange,
+    ]);
+
+    TrackedEventCount::factory()->create([
+        'type' => TrackedEventType::AiThreadSaved,
+    ]);
+
+    $totalExchanges = TrackedEventCount::where('type', TrackedEventType::AiExchange)->value('count');
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['ai_exchanges'])->toBe($totalExchanges);
+});
+
+it('checks the API returns Saved AI Chats', function () {
+    $randomRecords = random_int(1, 10);
+
+    AiThread::factory()->count($randomRecords)->saved()->create();
+
+    $softDeleteAiThread = AiThread::factory()->saved()->create();
+    $softDeleteAiThread->delete();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['saved_ai_chats'])->toBe($randomRecords);
+});
+
+it('checks the API returns Saved Prompts', function () {
+    $randomRecords = random_int(1, 10);
+
+    Prompt::factory()->count($randomRecords)->create();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['saved_prompts'])->toBe($randomRecords);
+});
+
+it('checks the API returns Prompts Inserted', function () {
+    $randomRecords = random_int(1, 10);
+
+    PromptUse::factory()->count($randomRecords)->create();
+
+    $softDeletePromptUse = PromptUse::factory()->create();
+    $softDeletePromptUse->delete();
+
+    $response = get('/api/utilization-metrics');
+
+    $data = $response->json('data');
+
+    $response->assertStatus(200);
+
+    expect($data['prompts_inserted'])->toBe($randomRecords);
+});
+
+it('checks the API returns Retention CRM Users', function () {
+    $randomRecords = random_int(1, 10);
+
+    $licenseSettings = app(LicenseSettings::class);
+
+    $licenseSettings->data->limits->retentionCrmSeats = $randomRecords + 1;
+    $licenseSettings->save();
+
+    User::factory()->count($randomRecords)->licensed([LicenseType::RetentionCrm])->create();
+
+    $softDeleteRetentionCrmUser = User::factory()->licensed([LicenseType::RetentionCrm])->create();
+    $softDeleteRetentionCrmUser->delete();
+
+    $response = get('/api/utilization-metrics');
```

Comment:
We should be using the route name here, not calling the path directly.

Reply:
Done